fbfmaproom/fbfmaproom.py

```diff
@@ -383,6 +383,8 @@ def get_mpoly(mode, country_key, geom_key):
         mpolygon = MultiPolygon([Polygon([(x0, y0), (x0, y1), (x1, y1), (x1, y0)])])
     else:
         label, mpolygon = retrieve_geometry2(country_key, int(mode), geom_key)
+        if isinstance(mpolygon, Polygon):
```

Comment:
Something that has been vexing me is why this is necessary at all, and why Madagascar works without it. What's weird is that there's two different pathways that shapes go through, and the first already did something like this, but the second (this one) did not

Reply:
Well let me clarify: "mechanically" I know why it's necessary, it's because Polygons aren't iterable (of course). However, I just don't understand what's different about this circumstance such that we get a MultiPolygon from the query in all other circumstances, including another "computed" top-level boundary (in the case of Madagascar)

Reply:
Yeah, I don't know either. But we had this check elsewhere already (see retrieve_vulneraiblity), so it's not just Oromia.

Reply:
Yeah I noticed that too. I'm very curious why this is only showing up now but I suppose it doesn't matter